ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/BlockTree.hs

```diff
@@ -149,19 +150,17 @@ findPath source target blockTree = do
 --                      ╰─────3──4─────5
 --
 -- Returns a list of strings intended to be catenated with a newline.
-prettyPrint :: AF.HasHeader blk => BlockTree blk -> [String]
+prettyPrint :: forall blk . AF.HasHeader blk => BlockTree blk -> [String]
```

Comment:
I see we both generalised this. The techniques are different but the result is the same, I believe. I personally don't care as long as it is generalised!